src/components/LobbyPlayerIndicator.vue

```diff
@@ -15,12 +15,13 @@
       class="ready-overlay text-truncate"
       :style="{ 'font-size': readyFontSize, 'padding-bottom': readyPadding }"
     >
-      READY
+      {{ t('lobby.ready') }}
```

Comment:
I wonder if we should do the caps via css, would avoid the need to capitalize or case in any specific way in the locale files.

Reply:
Yeah it's an interesting question especially considering that some pages will want upper vs lower vs first letter upper cased. In reality the buttons already have default Vuetify css to uppercase them per material design spec so in that sense we could do whatever we want for button text.